packages/grid/_modules_/grid/components/cell/GridActionsCell.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import MenuList from '@material-ui/core/MenuList';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+// @ts-expect-error fixed in Material-UI v5, types definitions were added.
+import { unstable_useId as useId } from '@material-ui/core/utils';
+import { GridRenderCellParams } from '../../models/params/gridCellParams';
+import { gridClasses } from '../../gridClasses';
+import { GridMenu } from '../menu/GridMenu';
+import { GridActionsColDef } from '../../models/colDef/gridColDef';
+
+const hasActions = (colDef): colDef is GridActionsColDef => typeof colDef.getActions === 'function';
+
+export const GridActionsCell = (props: GridRenderCellParams) => {
+  const [anchorEl, setAnchorEl] = React.useState(null);
+  const menuId = useId();
+  const buttonId = useId();
+  const { colDef, id, api } = props;
+
+  if (!hasActions(colDef)) {
+    throw new Error('Material-UI: Missing the `getActions` property in the `GridColDef`.');
+  }
+
+  const showMenu = (event) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const hideMenu = () => {
+    setAnchorEl(null);
+  };
+
+  const options = colDef.getActions(api.getRowParams(id));
+  const iconButtons = options.filter((option) => !option.props.showInMenu);
+  const menuButtons = options.filter((option) => option.props.showInMenu);
+
+  return (
+    <div className={gridClasses.actionsCell}>
+      {iconButtons.map((button, index) => React.cloneElement(button, { key: index }))}
+      {menuButtons.length > 0 && (
+        <IconButton
+          id={buttonId}
+          aria-label={api.getLocaleText('actionsCellMore')}
+          aria-controls={menuId}
+          aria-expanded={anchorEl ? 'true' : undefined}
+          aria-haspopup="true"
+          size="small"
+          onClick={showMenu}
+        >
+          <MoreVertIcon fontSize="small" />
+        </IconButton>
+      )}
+      {menuButtons.length > 0 && (
+        <GridMenu
+          id={menuId}
+          onClickAway={hideMenu}
+          onClick={hideMenu}
+          open={Boolean(anchorEl)}
+          target={anchorEl}
+          position="bottom"
+          aria-labelledby={buttonId}
+        >
```

Comment:
Are we sure about the menu placement relative to the target?

<img width="243" alt="Screenshot 2021-09-05 at 12 53 57" src="https://user-images.githubusercontent.com/3165635/132124223-54dc5b0d-5ca3-4e99-94c8-d6690f1634d8.png">

I was personally expecting something similar to Stripe:

<img width="226" alt="Screenshot 2021-09-05 at 12 55 53" src="https://user-images.githubusercontent.com/3165635/132124249-5c2edbea-e211-428a-850a-e72aef51f469.png">

Reply:
I changed it to `bottom-end` and added a position prop to allow users to change it (the actions could be the first column).